Source/WebCore/page/Page.cpp

```diff
@@ -3759,13 +3759,17 @@ void Page::recomputeTextAutoSizingInAllFrames()
 
 #endif
 
-bool Page::acceleratedFiltersEnabled() const
+FilterMode Page::preferredFilterMode() const
```

Comment:
I guess this is OK for now, but at some point we might want to have both "native" and Core Image filters enabled, using native when possible, falling back to Core Image, and then falling back to software, depending on the filter graph.